app/resources/base.html.template

```diff
@@ -64,6 +66,9 @@
         {% endwith %}
       {% endif %}
     </style>
+    <!-- AMP Analytics -->
```

Comment:
Use `{# AMP Analytics #}` instead. Then it's removed from the generated HTML.

Reply:
done